### PR TITLE
return the appliance address if db is None

### PR DESCRIPTION
**Pull request by amavinag**
* this will avoid to NoneType Error Exception.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit beacf6e22c4fc23f8e243c0257c17440cc5eeeff:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by psav** [comment]
@mfalesni @jkrocil I emailed you about this one

**Comment by dajoRH** [comment]
Lint report for commit ed3fcd9d6e8c6bf6fea6cb27cf9b5113263f76dd:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e971002bc70a4d19a899b2c60d42ba9e098f20de:

Everything seems all right :smile: .
*CFME QE Bot*

File: utils/appliance.py [hunk 1]
```diff
@@ -1356,16 +1356,34 @@ def wait_for_ssh(self, timeout=600):
...
+
     @cached_property
     def db_address(self):
         # pulls the db address from the appliance by default, falling back to the appliance
         # ip address (and issuing a warning) if that fails. methods that set up the internal
         # db should set db_address to something else when they do that
         try:
             if self.version >= '5.6':
-                db = self.get_yaml_config("vmdb")['server']['host']
+                db = self.wait_for_host_address('vmdb')
```
Comment (psav) [inline utils/appliance.py]: Really like the abstraction up the chain here ;)

**Comment by psav** [comment]
Is being reraised against master

**Pull request closed without merging**